sdk/quantum/azure-quantum-jobs/CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
+
+### Added
+- Initial preview of Azure.Quantum.Jobs
```

Comment:
This should include text to indicate that this is an initial release and provide a link to the getting started section of README.md